lib/extras/codec_test.cc

```diff
@@ -396,6 +406,78 @@ TEST(CodecTest, TestWideGamut) {
 
 TEST(CodecTest, TestPNM) { TestCodecPNM(); }
 
+TEST(CodecTest, FormatNegociation) {
+  const std::vector<JxlPixelFormat> accepted_formats = {
+      {.num_channels = 4,
```

Comment:
I don't think this is standard C++11... (IIRC it's either C++17 or a GNU extension)

Reply:
Oh, right, I keep forgetting. Thanks.